### ISSUE TITLE
Support relative paths in docker-compose 'volumes:' statements

### ISSUE THREAD

**Issue by emetsger** [fabric8io/docker-maven-plugin#846]
### Description
Relative paths in docker-compose `volumes:` statements are not supported.  For example, the volumes stanza below 
```
version: '2.2'
services:
  solr:
  ...
     volumes:
     - "./cores:/opt/solr/server/solr/mycores"
```
results in the following error on `docker:start`:
```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.21-SNAPSHOT:start (default-cli) on project rmap-indexing-solr: I/O Error: Unable to create container for [solr:6.6.0-alpine] : create ./cores: "./cores" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed (Internal Server Error: 500) -> [Help 1]
```

### Info
* d-m-p version : `e50a7a5`, `0.21-SNAPSHOT`
* Maven version (`mvn -v`) :
```
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T10:41:47-06:00)
Maven home: /Users/esm/maven-3
Java version: 1.8.0_131, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.6", arch: "x86_64", family: "mac"
```
* Docker version : `Docker version 1.11.2, build b9f10c9`, `docker-machine version 0.7.0, build a650a40`

**Comment by emetsger** [comment]
FYI, after upgrading the docker instance to `v17.06.1-ce`, the error message is a little more clear, with the additional text: `If you intended to pass a host directory, use absolute path`

```
DOCKER> I/O Error [Unable to create container for [solr:6.6.0-alpine] : create ./cores: "./cores" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path (Internal Server Error: 500)]
```

**Comment by emetsger** [comment]
Closed by merge `802fa9b` of pr #848
